prow/plugins/internal/regex-based-label-match/fake.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package regexbasedlabelmatch
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/github"
+)
+
+type FakeGitHub struct {
+	Labels                               sets.String
+	IssueLabelsAdded, IssueLabelsRemoved sets.String
+	Commented                            bool
+}
+
+func NewFakeGitHub(initialLabels ...string) *FakeGitHub {
```

Comment:
Oh I see that this FakeGithub was moved from require-matching-label-test. If you can use the existing fakegithub with minimal changes it would be nice, otherwise it is probably not in the scope of this PR to make that change.